dev/MRTCore/TestApps/winui_desktop_packaged_app/winui_desktop_packaged_app.sln

```diff
@@ -31,8 +31,8 @@ Global
 		{28B258CA-5ECC-41EE-A5B0-C2F24F3A4984}.Release|x86.ActiveCfg = Release|x86
 		{28B258CA-5ECC-41EE-A5B0-C2F24F3A4984}.Release|x86.Build.0 = Release|x86
 		{28B258CA-5ECC-41EE-A5B0-C2F24F3A4984}.Release|x86.Deploy.0 = Release|x86
-		{AF3AA2D2-7150-4996-8C38-0AB44604C3D0}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
-		{AF3AA2D2-7150-4996-8C38-0AB44604C3D0}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{AF3AA2D2-7150-4996-8C38-0AB44604C3D0}.Debug|Any CPU.ActiveCfg = Debug|x86
```

Comment:
VS auto-made that change after I cleaned my repo and it complained that there was some inconsistency in the project file.